ext/random/engine_mt19937.c

```diff
@@ -392,11 +392,11 @@ PHP_METHOD(Random_Engine_Mt19937, __debugInfo)
 
 	if (engine->engine.algo->serialize) {
 		array_init(&t);
+		zend_hash_str_add(Z_ARR_P(return_value), "__states", strlen("__states"), &t);
```

Comment:
if _you think_ there is an actual memory leak, please do a separated PR.
It is also better to create a branch from an appropriate origin branch.